portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+ <head>
+     <meta charset="UTF-8">
+     <title>Gallery</title>
+     <link rel="stylesheet" href="style.css">
+     <script src="script.js"></script>
+     <style>
+     body {
+        background: url(images/better.jpg) no-repeat center center fixed;
+     }
+
+     html {
+        scroll-behavior: smooth;
+     }
+     </style>
+ </head>
+ <body>
+    <div id="content">
+        <div style="border-bottom:3px solid #000">
+            <a href="index.html">
+                <img src="/images/home.webp" width="50" height="50" alt="Home">
+            </a>
+        </div>
+        
+        <div id="bold"><h2>Some Places I've Been To<h2></div>
+
+        <img src="/images/toledo.webp" width="400" height="221" alt="Toledo-Spain">
```

Comment:
The benefit of sizing these with CSS (as commented above), is that all these images are supposed to be the same size. If you give all of these images a class which CSS targets to set the width and height, then if you ever needed to change the size, you can change the one CSS rule, rather than having to hunt down and change every image